Automattic/jetpack
# Try to fix failing recommendations.test.js and waf-blocking.test.js E2E tests

"Jetpack post-connection e2e tests" have been failing since forever:

https://github.com/Automattic/jetpack/actions/workflows/e2e-tests.yml

e.g. here for https://github.com/Automattic/jetpack/pull/34739:

https://github.com/Automattic/jetpack/actions/runs/7348501314/job/20006837157?pr=34739

As per https://github.com/Automattic/jetpack/pull/34796#pullrequestreview-1798540385, I've updated XPath selectors that check for the plan in the rendered webpage.

@jboland88, @ArSn, you've last touched this code, could you maybe have a look? I'm not familiar with those tests myself.

## Proposed changes:

See above.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Make sure the tests succeed.
* Make sure what needs to get tested still gets tested.
